src/client/components/Checker.tsx

```diff
@@ -49,6 +49,19 @@ export const Checker: FC<CheckerProps> = ({ config }) => {
   const googleAnalytics = useGoogleAnalytics()
   const outcomes = useRef<HTMLDivElement | null>(null)
 
+  // conditionally toggle description label between center align (1 line) and left align (> 1 line)
+  const descriptionRef = useRef<HTMLParagraphElement>(null)
```

Comment:
Could we just set `maxWidth` to `100%` for subtitle with no `text-align`? If the text is less than one line, the width will be auto to fit the content. If it is more than 1 line, it will be width `100%`.

![image](https://user-images.githubusercontent.com/3666479/120995123-b1270880-c7b7-11eb-8602-73e950c26cd5.png)

Reply:
Hi, the reason why I had to opt for this particular method of toggling the width was because of newline characters, which could wrap the subtitle onto a new line without necessarily increasing the width to 100%. As such, it would wrap in weird ways like this:

![Screenshot 2021-06-07 at 8 35 25 PM](https://user-images.githubusercontent.com/21305518/121017477-e63f5500-c7cf-11eb-808a-6ded7c16dbbc.png)

As such, this snippet would ensure that the width would always be increased to 100% if the subtitle exceeds one line. However, I do feel that this function is slightly overkill for this sort of formatting so let me know if you have a cleaner solution for this, thanks!

Reply:
I think this is quite an edge case though. Might be more consistent to just use the default CSS layout. I have not tested, but I think if we resize the window, this wouldn't re-evaluate. Would prefer that we not handle this for now. We can always revisit this if users raise an issue about this rather than pre-optimising for this. The solution could also be a design change rather than a code one.